tests/global_discovery_catalogue/test_gdc_functional.py

```diff
@@ -142,7 +142,7 @@ def _publish_wcmp2_trigger_broker_message(wcmp2_file, cache_a_wis2=None) -> None
     trigger_client = subscribe_trigger_client()
     trigger_client.pub('config/a/wis2/metadata-pub', json.dumps(message))
 
-
+test_3
```

Comment:
Remove

Reply:
I do not understand the change request.  Removing "client.tls_set(**tls_settings)" will prevent the TLS settings from being applied.

`    if connection_info.port in [443, 8883]:`
`        tls_settings = { 'tls_version': 2 }`
`        tls_settings['cert_reqs'] = ssl.CERT_NONE`
`    client.tls_set(**tls_settings)`
`